master/lustrefactory.py

```diff
@@ -6,26 +6,32 @@
 from buildbot.steps.shell import ShellCommand
 from buildbot.steps.shell import Configure
 from buildbot.steps.shell import SetPropertyFromCommand 
-from buildbot.steps.transfer import FileUpload
+from buildbot.steps.transfer import FileUpload, FileDownload
 from buildbot.steps.trigger import Trigger
 from buildbot.status.results import SUCCESS 
 from buildbot.status.results import FAILURE 
 from buildbot.status.results import SKIPPED 
 from buildbot.status.results import WARNINGS
 
-def do_step_build(step, name):
+def do_step_if_value(step, name, value):
     props = step.build.getProperties()
-    if props.hasProperty(name) and props[name] == "yes":
+    if props.hasProperty(name) and props[name] == value:
         return True
     else:
         return False
 
+def do_step_build(step, name):
+    return do_step_if_value(step, name, "yes")
```

Comment:
Ah, I see.   Technically, there are others ways to accomplish that (lambda), but I agree, that would probably be uglier.

But yes, I think it would be good to eliminate do_step_build and use the one do_step_if_value directly.  One less level of hopping I have to do in the code is appreciated.